{{cookiecutter.stack_name}}/.github/workflows/docker.yml

```diff
@@ -56,6 +56,7 @@ jobs:
         run: python -m pytest tests
 
       - name: Login to Docker Hub
+        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main'
```

Comment:
Excellent fix. Note below though, there is a if statement on when to push that should be updated to have `main` as well.

Reply:
Nice catch, fixed here as well.